src/quo/components/inputs/address_input/style.cljs

```diff
@@ -35,7 +34,8 @@
          :color        (colors/theme-colors colors/neutral-100 colors/white theme)
          :margin-top   (if platform/ios? 0 -4)
          :margin-right 8
-         :height       (if platform/ios? 24 40)))
+         :max-height   50
+         :padding      0))
```

Comment:
@OmarBasem Did you check how it is looking on Android? I'm asking because we used different numbers for each platform before. It'd be great if you add an screenshot (or video) for Android too :+1:

Reply:
@ulisesmac the video is for Android :)

Reply:
You're right, thanks!